kornia/enhance/adjust.py

```diff
@@ -595,15 +595,18 @@ def adjust_log(image: Tensor, gain: float = 1, inv: bool = False, clip_output: b
     return img_adjust
 
 
-def _solarize(input: Tensor, thresholds: Union[float, Tensor] = 0.5) -> Tensor:
+def _solarize(input: Tensor, thresholds: Union[float, Tensor] = 0.5, relaxation: Optional[str] = None) -> Tensor:
```

Comment:
```suggestion
def _solarize(input: Tensor, thresholds: Union[float, Tensor] = 0.5, relaxation: str | None = None) -> Tensor:
```
then `from __future__ import annotations`

Reply:
done